### DISCUSSION TITLE
scheduled share

### DISCUSSION THREAD

**Discussion by scratch-eeveeg** [ScratchAddons/ScratchAddons#965] [Ideas] [answered]
this might already be suggested, or denied for whatever reason, but what about being able to schedule a project to share? like how you can schedule an email to send, but with scratch projects, so that, for example, if you made Christmas presents for scratchers but won't be there on Christmas, you can schedule it to share at a certain time

**Comment by WorldLanguages** [comment, marked as answer]
Issue is, that wouldn't work unless you leave Chrome open.

**Reply by scratch-eeveeg** [reply]
even if its connected to the world clock and saved onto your account?

**Reply by pufferfish101007** [reply]
There are no scratch addons accounts, and it would be impossible to connect it to your scratch account. In addition, there'd have to be an external server, which we don't want.

**Reply by mxmou** [reply]
> even if its connected to the world clock and saved onto your account?

The only alternative would be to have a server log in to your Scratch account at that time and share the project, which would probably be against Scratch's ToU.

**Reply by scratch-eeveeg** [reply]
That makes sense
